### ISSUE TITLE
Question: are extra steps required to get correct log levels to show?

### ISSUE THREAD

**Issue by aecorredor** [kioannou/winston-datadog-logger#7]
So I got the logger to work fine, but I am seeing all logs as `info` in the datadog dashboard. From what I'm reading here https://docs.datadoghq.com/logs/faq/why-do-my-logs-show-up-with-an-info-status-even-for-warnings-or-errors/ this seems to be expected.

But, then I see this `eventMapping` option in the package which I assume does something similar?

My question is if we need to follow the steps to extract the status with a parser or if this library is supposed to do that automatically?

Thanks for the package!

**Comment by aecorredor** [comment]
Also, why does `silent` default to `true` for the transport? What happens if we set that to `false`?

**Comment by kioannou** [comment]
@aecorredor you can map the winston events to the datadog ones like this durint the initialization: 
```
  eventMapping: {
        debug: DogapiEvent.Info,
        error: DogapiEvent.Error,
        info: DogapiEvent.Info,
        silly: DogapiEvent.Info,
        verbose: DogapiEvent.Info,
        warn: DogapiEvent.Warning,
      },
```
have you done that? Keep in mind that this is the default configuration, so if you try to log with the event statuses `debug`, `info`, `silly`, or`verbose` they will end up like `info` in datadog.

Anyways I don't think that you should extract the status (you can try it). In my apps the library did that automatically.

`silent` is not used anymore.

**Comment by aecorredor** [comment]
Weird, I did exactly that. I’m deploying to zeit now v2, which uses lambdas under the hood. I’m not sure if it’s something related to that, but it shouldn’t since this just sends logs to datadog directly as an http request right?

What happens if `logDatadogEvents` is set to false?

**Comment by kioannou** [comment]
@aecorredor Exactly. You are right. I am not aware of zeit now v2. When I have time I will look at it.

 `logDatadogEvents` logs the responses or errors from datadog.
